### PR TITLE
Connector Tutorial - DOCS-5746 fix

### PR DESCRIPTION
**Pull request by george-hoffer**

### CODE REVIEW

**Review by dhirsch1 — APPROVED** [review]

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -121,4 +118,3 @@ xref:sap/sap-successfactors-connector.adoc[SAP SuccessFactors]
 == See Also
 
 * https://forums.mulesoft.com[MuleSoft Forum]
-* https://support.mulesoft.com/s/knowledge[Knowledge Base Articles]
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/index.adoc:124]: Why remove the KB articles?

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -3,12 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Connectors facilitate integrations with third-party APIs. Modules are extensions to the Mule runtime that you can use when building a Mule application.
+Anypoint Connectors facilitate integrations with third-party APIs. Modules are extensions to the Mule runtime that you can use when building a Mule application. For a tutorial on how to use connectors, see the xref:4.2@::mule-runtime/mule-app-tutorial.adoc[Mule App Development Tutorial].
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/index.adoc]: The tutorial isn't for how to use connectors, it's for how to build an integration (and includes info about using connectors).  Maybe something like "The Mule app tutorial includes a section on adding the database connector to a sample Mule app" or something similar? Or you can just delete this sentence and add a link to the See Also section:
Mule App Tutorial (adding a database connector) or something. 

Can we add the relevant conceptual info from the old tutorial here?

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/index.adoc:124]: https://support.mulesoft.com/s/knowledge has been replace by https://help.mulesoft.com that covers much more than KB articles. I assume customers know about the Help Center.

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (george-hoffer) [reply modules/ROOT/pages/index.adoc]: Changed to:
See xref:4.2@::mule-runtime/mule-app-tutorial.adoc[Mule App Development Tutorial] for information on adding a connector to a sample Mule app.

Diane is writing a new connector intro that handles intro material.

**Pull request merged**